README.md

```diff
@@ -1,173 +1,67 @@
-
-Navigate to the USAspending API's main folder:
-
-        $ cd usaspending-api
-
-
-Install the dependencies.
-
-        $ pip install -r requirements/requirements.txt
-
-Install caching dependencies. (This is optional, and should only be done if you have installed Memcached and require AWS caching for development)
-
-        $ pip install -r requirements/caching_requirements.txt
-
-Next, configure your local settings. The `settings.py` file will check for an environment variable named `DATABASE_URL`. You can export this variable on the command line, or use a library like [autoenv](https://github.com/kennethreitz/autoenv) to load your environment variables when you `cd` into that directory.
-
-Exporting your `DATABASE_URL` environment variable should look something like this:
-
-        $ export DATABASE_URL='postgres://USER:PASSWORD@HOST:PORT/DATABASENAME'
-
-
-To test your database connection, try running the migrations that come with the project to set up your tables.
-
-        $ python manage.py migrate
-
-You should see output that looks like this:
-
-    (usaspending-api):~/usaspending-api$ ./manage.py migrate
-    Operations to perform:
-    Apply all migrations: admin, auth, awards, contenttypes, sessions
-    Running migrations:
-    Applying awards.0001_initial... OK
-
-Once you've done that, you're ready to start the development server
-
-    $ python manage.py runserver
-
-This will run the application at `127.0.0.1:8000` by default. You can change that by passing the host and port number in to the `runserver` command.
-
-you should see something like this:
-
-    Performing system checks...
-    System check identified no issues (0 silenced).
-    August 22, 2017 - 15:55:33
-    Django version 1.11.4, using settings 'usaspending_api.settings'
-    Starting development server at http://127.0.0.1:8000/
-    Quit the server with CONTROL-C.
-
-Now, go to `http://localhost:8000/api/v1/awards/` to see the API! There aren't any other urls set up at the moment.
-
-## Running the Test Suite
-
-To run the test suite and confirm that everything is working as expected, make sure you're in the top-level project directory and that your `DATABASE_URL` environment variable is set, and
-run the following from the command line:
-
-
-        py.test
-
-The tests will not run against your database in `$DATABASE_URL`, but against a
-temporary database named `test_(your db name)`, which they will create.
-
-## ETL Setup
-
-The Django server has extract, transform, and load (ETL) processes built in as custom management commands. To use them, follow these setup steps:
-
-  1. Configure the `DATA_BROKER_DATABASE_URL` environment variable. This is the same process as setting up the API backend. Exporting `DATA_BROKER_DATABASE_URL` should look like this:
-
-  ```
-  $ export DATA_BROKER_DATABASE_URL='postgres://USER:PASSWORD@HOST:PORT/DATABASENAME'
-  ```
+Available API endpoints may be found at `http://localhost:8000/docs/endpoints`
```

Comment:
@justinle In addition to pointing to where a developer could find API docs on their local env, can you also provide the link to the latest published production API docs. That would be:

https://api.usaspending.gov/

_(from there, they should be able to click the "Endpoints" link on the left nav to get further into the api docs)_

Reply:
The main endpoints link now points to the main API site, the local environment is also noted under the main link.